pkgs/os-specific/linux/kernel/common-config.nix

```
@@ -868,11 +868,13 @@ let
     };
 
     zram = {
-      ZRAM           = module;
-      ZRAM_WRITEBACK = option yes;
-      ZSWAP          = option yes;
-      ZPOOL          = yes;
-      ZBUD           = option yes;
+      ZRAM                          = module;
+      ZRAM_WRITEBACK                = option yes;
+      ZRAM_DEF_COMP_ZSTD            = whenAtLeast "5.11" yes;
+      ZSWAP                         = option yes;
+      ZSWAP_COMPRESSOR_DEFAULT_ZSTD = whenAtLeast "5.7" yes;
+      ZPOOL                         = yes;
+      ZBUD                          = option yes;
```

```suggestion
      Z3FOLD                          = option yes;
```
Forgot to mention, but I frequently get >3x compression ratios with Zstandard compressed `zram` or `zswap`.
Wich means `ZBUD` (limited to 2x compression ratios) is a bottleneck.

Also, I just discovered [`ZRAM_MULTI_COMP`](https://www.kernel.org/doc/html/latest/admin-guide/blockdev/zram.html#recompression), which is super cool.

---

Hm, we should enable that.

---

I don't know about zram, but for zswap at least the default is zsmalloc which should be even better.